app/decorators/course_decorator.rb

```diff
@@ -162,4 +201,14 @@ def not_on_find
   def edit_vacancy_link
     h.link_to("Edit", h.vacancies_provider_recruitment_cycle_course_path(provider_code: object.provider_code, recruitment_cycle_year: object.recruitment_cycle_year, code: object.course_code), class: "govuk-link")
   end
+
+  def find_max(attribute)
+    subject_attributes = object.subjects.map do |s|
+      if s.attributes[attribute].present?
+        s.send(attribute).to_i
```

Comment:
`send` vs `__send__` ?

Reply:
https://stackoverflow.com/questions/4658269/ruby-send-vs-send

Don't think there is any issue with using 'send' in this instance but will change for consistency

Reply:
No idea just highlighting